docs/interpreter/livy.md

```diff
@@ -56,14 +56,14 @@ Example: `spark.driver.memory` to `livy.spark.driver.memory`
     <td>URL where livy server is running</td>
   </tr>
   <tr>
-    <td>zeppelin.livy.spark.maxResult</td>
+    <td>zeppelin.livy.spark.sql.maxResult</td>
```

Comment:
I love this, but it might break existing users. Perhaps we don't change this for now?